src/rag/memvid_pipeline.py

```
@@ -0,0 +1,109 @@
+"""
+Memvid-enhanced RAG pipeline for Maya
+"""
+
+import google.generativeai as genai
+from typing import List, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)